packages/studio-plugin/src/types/FileMetadata.ts

```diff
@@ -1,16 +1,11 @@
 import { ComponentMetadata } from "./ComponentMetadata";
 import { ErrorFileMetadata } from "./ErrorFileMetadata";
-import { ModuleMetadata } from "./ModuleMetadata";
 
 export enum FileMetadataKind {
   Component = "componentMetadata",
-  Module = "moduleMetadata",
   Error = "errorMetadata",
 }
 
-export type FileMetadata =
-  | ComponentMetadata
-  | ModuleMetadata
-  | ErrorFileMetadata;
+export type FileMetadata = ComponentMetadata | ErrorFileMetadata;
 
 export type ValidFileMetadata = Exclude<FileMetadata, ErrorFileMetadata>;
```

Comment:
Do we want to keep this type?  I feel like it is nice conceptually, but it's really just `ComponentMetadata`.

Reply:
I think I'd prefer simplifying things as much as we can for now. Down the road, if we end up supporting other types of file metadata, then we can see about adding back a type like this. In fact, what do you think about renaming `FileMetadata`, or removing it entirely and just having `ComponentMetadata` and `ErrorComponentMetadata` like we have for `PageState`?

Reply:
Actually, could I just consolidate `ComponentMetadata` and `ValidFileMetadata` as `ComponentMetadata` in this PR for now?  I think the FileMetadata naming pattern goes pretty deep, and I'm not exactly sure what we would want to call it instead.

Reply:
Makes sense. Can you make an item to investigate?